codegen/src/main/java/io/improbable/keanu/codegen/python/KeanuProjectDoclet.java

```diff
@@ -7,6 +7,7 @@
 import com.sun.javadoc.RootDoc;
 import com.sun.tools.doclets.standard.Standard;
```

Comment:
This causes issues building with some open source JDKs according to Chris M - perhaps we want to document that building Keanu depends on the Oracle JDK or something?